### PR TITLE
Feature Icons page performance improvement and prepare icons to beta

### PR DESCRIPTION
**Pull request by amorimjj**

### CODE REVIEW

File: packages/ignition/src/app/components/search/search.tsx [hunk 1]
```diff
@@ -1,18 +1,61 @@
...
 
-const SearchButton = ({ lang }: SearchButtonProps) => {
+const SearchAsInput = ({ placeholder }: SearchAsButtonProps) => (
+  <Button
+    className="w-72 m text-right bg-transparent focus:outline-none text-slate-400 dark:text-slate-300"
+    onClick={() => open()}
+  >
+    <div className="inline-block p-1 border-b border-slate-800 dark:border-white">
+      <span className="text-left text-sm w-40 inline-block">{`${placeholder}`}</span>
+      <kbd className="font-sans font-semibold text-slate-400 text-xs dark:text-slate-500">
```
Comment (programad) [inline packages/ignition/src/app/components/search/search.tsx:46]: Tiny duplicated code.

File: packages/ignition/src/app/error.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"use client";
+import Logo from "@/components/logo";
+import OnError from "./on-error";
+
+const Error = () => (
```
Comment (programad) [inline packages/ignition/src/app/error.tsx:5]: Maybe there is something funny going on with the error page? 

This is how it rendered for me in Opera/Chrome.

image

File: packages/ignition/src/mdx-components.tsx [hunk 3]
```diff
@@ -67,7 +66,7 @@ export function useMDXComponents(components: MDXComponents): MDXComponents {
     ),
     code: ({ children, ...props }: CodeProps) => (
       <CodeContainer {...props}>
-        <Code>{children}</Code>
+        <Block>{children}</Block>
```
Comment (programad) [inline packages/ignition/src/mdx-components.tsx:69]: In a future update, I think this should be called CodeBlock to match the parent naming style "CodeContainer"

**Review by programad — APPROVED** [review]
Just some minor issues to tackle on the next iteration.

Code 👍

File: packages/ignition/src/app/error.tsx (hunk 2 above)
Comment (amorimjj) [reply packages/ignition/src/app/error.tsx:5]: No, it's look right. We must working on that, but having just a white page as we have now it's terrible for the users, so I've putted this simple one on place

File: packages/ignition/src/mdx-components.tsx (hunk 3 above)
Comment (amorimjj) [reply packages/ignition/src/mdx-components.tsx:69]: Sure

**Pull request merged**